Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/CommonComponents/AdditionalFieldsWindow.tsx

```diff
@@ -69,6 +69,8 @@ function AdditionalFieldsWindow(props: IProps): JSX.Element {
 
     const EmptyField: SystemCenter.Types.AdditionalField = { ID: 0, FieldName: '', Type: 'string', ParentTable: props.Type, ExternalDB: '', ExternalDBTable: '', ExternalDBTableKey: '', IsSecure: false, Searchable: false };
 
+    const [editNew, setEditNew] = React.useState<Application.Types.NewEdit>('New');
```

Comment:
Does this replace `edit`? If that's the intention let's remove the state for `edit`

Reply:
No, `edit` indicates if the page is in Edit mode (the user has clicked the `Edit` button), where `editNew` indicates whether the user is adding a new Field or editing an existing Field for the `Modal Title` (line 509).